python/llm/src/ipex_llm/transformers/kv.py

```diff
@@ -24,19 +24,28 @@
     init_fp8_kv_cache, append_fp8_kv_cache,
     init_kv_cache, append_kv_cache, extend_kv_cache
 )
-from typing import Optional, Dict, Tuple, Any
+from typing import Optional, Dict, Tuple, Any, List
 from transformers.cache_utils import DynamicCache
 from ipex_llm.utils.common.log4Error import invalidInputError
 
 
 class DynamicFp8Cache(DynamicCache):
+    def __init__(self, num_hidden_layers: Optional[int] = None) -> None:
+        super().__init__()
+        # fix transformers >= 4.45
```

Comment:
may also need to fix `DynamicCompressCache` and `DynamicCompressFp8Cache`

Reply:
fixed